bonfire/bonfire.py

```diff
@@ -742,6 +748,8 @@ def _cmd_namespace_extend(namespace, duration, local):
 @options(_timeout_option)
 def _cmd_namespace_wait_on_resources(namespace, timeout, db_only):
     """Wait for rolled out resources to be ready in namespace"""
+    if not namespace:
+        namespace = get_current_namespace()
```

Comment:
Same here?

Reply:
(I am referring to the possibility that `get_current_namespace` could return `None`)

Reply:
Yep, added.